godot/ModularWeapons/ModularWeapon/src/Projectile/Basic/BasicProjectile.gd

```diff
@@ -30,11 +29,15 @@ func _on_Timer_timeout() -> void:
 # @tags - virtual
 func _impact() -> void:
 	set_physics_process(false)
-	tween.interpolate_property(self, "modulate", modulate, modulate * 3, 0.1, Tween.TRANS_CUBIC)
-	tween.interpolate_property(self, "modulate", modulate * 3, Color.transparent, 0.2, 0, 2, 0.1)
-	tween.start()
-	yield(tween, "tween_all_completed")
-	queue_free()
+#	tween.interpolate_property(self, "modulate", modulate, modulate * 3, 0.1, Tween.TRANS_CUBIC)
+#	tween.interpolate_property(self, "modulate", modulate * 3, Color.TRANSPARENT, 0.2, 0, 2, 0.1)
+#	tween.start()
+#	await tween.tween_all_completed
+#	queue_free()
+	var tween = get_tree().create_tween().bind_node(self)
```

Comment:
Instead of using the tree `create_tween()` and binding the node we can just do it calling the node method instead: `var tween = create_tween()`